app/grandchallenge/core/templates/home.html

```diff
@@ -2,68 +2,120 @@
 {% load url %}
 {% load static %}
 {% load humanize %}
-{% load cache %}
-{% load round_to %}
 
 {% block content %}
```

Comment:
We could cache this entire section to reduce the number of times the database is hit for the latest challenges, algorithms and blog posts. I'm not sure how "expensive" these queries are. For the blog posts maybe it's nice if the content gets updated instantly though?

Reply:
Yes this absolutely should be cached. If the data are delayed by 10 minutes (or 1 hour) it doesn't really matter IMO.